codegen_java/project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.2.6
\ No newline at end of file
+sbt.version=1.4.9
```

Comment:
@breilly2 We discussed "should we change this" with BT-38 still unfinished and this codegen is not working for me

Reply:
FYI, while checking if rawls was using a scala version beyond `2.12.12` and had figured out a solution to [scoverage's publishing issues](http://www.dereferer.org/?https%3A%2F%2Fgithub%2Ecom%2Fscoverage%2Fsbt%2Dscoverage%2Fissues%2F319%23issuecomment%2D792068079), I stumbled upon this comment:

https://github.com/broadinstitute/rawls/blob/e5f42062220725e914297528f93964f4497fbe8e/project/Settings.scala#L74

ToL 1:
On the last day of last year's retreat someone [uploaded a "preview" scala 2.13 version](https://broadinstitute.jfrog.io/ui/repos/tree/General/libs-release-local%2Forg%2Fbroadinstitute%2Fcromwell%2Fcromwell-client_2.13%2F0.1-d334ac0b6-SNAP) of this `cromwell-client` library. No idea how far they got, and I assume this was a mini-hackathon in between retreat presentations. Anyway, if we wanted to continue towards shaving this particular yak we can perhaps beg that person for help/verification that sbt 1.4.9 Works For Them™ outside of CI.

ToL 2:

Upon closer inspection, I'm curious if this generated library even needs to be dependent on a scala library (2.11, 2.12, 2.13, etc.). The generated code is actually Java. The build tool is scala based and by default might be [adding the scala dependency](https://broadinstitute.jfrog.io/ui/repos/tree/PomView/libs-release-local%2Forg%2Fbroadinstitute%2Fcromwell%2Fcromwell-client_2.12%2F0.1-d334ac0b6-SNAP%2Fcromwell-client_2.12-0.1-d334ac0b6-SNAP.pom), but there's no actual dependencies on anything scala in the actual generated java code, afaict.